src/Content/HtmlResult.php

```diff
@@ -15,16 +15,9 @@
 
 class HtmlResult extends Result
 {
-    /**
-     * @var Environment
-     */
-    private $twig;
-
-    public function __construct(RequestStack $request, Environment $twig)
+    public function __construct(RequestStack $request, private Environment $twig)
```

Comment:
Constructor property promotion is only supported from PHP 8, because PHP 8 is the only version with active support, feel free to drop 7 in composer.json.